examples/LossMinimization/loss_minimization.jl

```diff
@@ -1,122 +1,217 @@
-using Distributions
-using LinearAlgebra
-using Random
-using Plots
-using EnsembleKalmanProcesses.EnsembleKalmanProcessModule
-using EnsembleKalmanProcesses.ParameterDistributionStorage
-
-# Seed for pseudo-random number generator for reproducibility
+# # Minimization of Loss function
+#
+# First we load the required packages.
+
+using
+    Distributions,
+    LinearAlgebra,
+    Random,
+    Plots
+
+using
+    EnsembleKalmanProcesses.EnsembleKalmanProcessModule,
+    EnsembleKalmanProcesses.ParameterDistributionStorage
+
+# ## Loss function with single minimum
+#
+# Here, we minimize the loss function
+# ```math
+# G₁(u) = \|u - u_*\| ,
+# ```
+# where ``u`` is a vector of parameters and ``u_*`` is a vector of optimal parameters, here
+# taken to be ``u_* = (-1, 1)``.
+#
+# We set the seed for pseudo-random number generator for reproducibility.
 rng_seed = 41
 Random.seed!(rng_seed)
+nothing # hide
 
-# Number of synthetic observations from G(u)
+# We choose the number of observations and noise level,
 n_obs = 1
-# Defining the observation noise level
-noise_level =  1e-8   
-# Independent noise for synthetic observations       
-Γy = noise_level * Matrix(I, n_obs, n_obs) 
-noise = MvNormal(zeros(n_obs), Γy)
-
-# Loss Function (unique minimum)
-function G(u)
-    return [sqrt((u[1]-1)^2 + (u[2]+1)^2)]
-end
+noise_level = 1e-3
+nothing # hide
+
+# Independent noise for synthetic observation:
+Γ_stabilisation = noise_level * Matrix(I, n_obs, n_obs) 
+noise = MvNormal(zeros(n_obs), Γ_stabilisation)
```

Comment:
```suggestion
noise = MvNormal(zeros(N_observations), Γ_stabilisation)
```

Reply:
I don't think `noise` is used anywhere in this file; therefore these lines are irrelevant to the example. @odunbar is this a typo / mistake?